src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -828,9 +833,9 @@ internal async Task Cleanup(bool async, bool connectionClosing=false)
             }
         }
 
-        #endregion
+#endregion
```

Comment:
Any reason for this? In general in Npgsql we indent regions in line with the code... Let's revert this...

Reply:
Not me, but IDE. Fixed.